dhcp_server/config.yaml

```
@@ -1,5 +1,5 @@
 ---
-version: 1.4.0
+version: 1.5.0
 slug: dhcp_server
 name: DHCP server
```

Maybe this adds to the clarity?

```suggestion
name: DHCP server (DEPRECATED)
```

---

Sounds good!